kitchen-tests/Gemfile

```diff
@@ -2,9 +2,9 @@ source "https://rubygems.org"
 
 group :end_to_end do
   gem 'berkshelf'
-  gem 'test-kitchen', '~> 1.4.0'
+  gem 'test-kitchen', '~> 1.4.0', github: 'test-kitchen/test-kitchen'
```

Comment:
@jkeiser the TK changes got released as 1.4.2, so you can depend on that instead of master